test/javascripts/acceptance/new-topic-dropdown-test.js

```diff
@@ -0,0 +1,37 @@
+import { getOwner } from "@ember/owner";
+import { visit } from "@ember/test-helpers";
+import { test } from "qunit";
+import { acceptance, resetSite } from "discourse/tests/helpers/qunit-helpers";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+
+acceptance(
+  "DiscoursePresetTopicComposer - new topic dropdown",
+  function (needs) {
+    needs.user();
+
+    needs.hooks.beforeEach(() => {
+      resetSite({
+        topic_preset_buttons: [{ id: "NEW_BUG", name: "Report a bug" }],
+      });
+    });
+
+    test("app event on change", async function (assert) {
+      let triggered = false;
+      const appEvents = getOwner(this).lookup("service:app-events");
+      appEvents.on(
+        "discourse-preset-topic-composer:new-topic-preset-selected",
+        () => {
+          triggered = true;
```

Comment:
instead of a var you can use

```suggestion
          assert.step("triggered");
```